src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/QueryableExtensions.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace System.Linq;
+
+public static class QueryableExtensions
+{
+    public static async Task<PaginatedList<TEntity>> GetPaginatedListAsync<TEntity>(
+        this IQueryable<TEntity> queryable,
+        Expression<Func<TEntity, bool>> predicate,
+        PaginatedOptions options,
+        CancellationToken cancellationToken = default) where TEntity : class, IEntity
+    {
+        var result = await GetPaginatedListAsync(
+            queryable,
+            predicate,
+            (options.Page - 1) * options.PageSize,
```

Comment:
I don't think Page needs -1, just PageIndex need it.

Reply:
The GetPaginatedListAsync method of Masa Framework also uses Page and - 1, which is consistent with it

Reply:
Fix Masa Framework first.

Reply:
Page represents which page PageIndex is the index in Pages

Reply:
Yes.

Reply:
OK